config/environments/test.rb

```diff
@@ -45,4 +45,6 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.active_job.queue_adapter = :test
+
+  config.hosts << 'submitter-app'
```

Comment:
i think just call `config.hosts.clear` for dev and test like you did for the other ones

Reply:
Done!